documents/design/fine_tuning_teb_parameters.md

```diff
@@ -0,0 +1,39 @@
+# Fine Tuning TEB Parameters
+
+*Issue #628*
+
+**Author:**
+- Amy Yao
+
+## The Problem
+
+The problem I am trying to solve is to fine tune the timed-elastic-bands control parameters for the
+robot.
+
+## Proposed Solution
+
+- I will first watch more videos and read more articles abut timed-elastic-bands control to make sure
+  that I understand how TEB works. I will learn what the appropriate values for each paramater
+  are and what will changing each of the variables do.
+- Then, I will try to use rviz to adjust, test, and tune the parameters before I make changes to the file.
+- If all the parameters seem to be tuned, I will then make changes to the values in the file, and test
+  on the robot if possible.
+
+## Questions & Research
+
+I need to research more about model predictive control and timed-elastic-bands in order to solve this issue,
+and I also need to read more about teb_local_planner on ROS Wiki.
```

Comment:
I think it would be nice to make a preliminary list of resources that you will be using in order to learn more.